src/Reflection/ReflectionClass.php

```diff
@@ -902,7 +902,7 @@ public function getProperties(int $filter = 0): array
                     [],
                     $this->getParentClass()?->getProperties(ReflectionPropertyAdapter::IS_PUBLIC | ReflectionPropertyAdapter::IS_PROTECTED) ?? [],
                     ...array_map(
-                        static fn (ReflectionClass $ancestor): array => $ancestor->getProperties(ReflectionPropertyAdapter::IS_PUBLIC | ReflectionPropertyAdapter::IS_PROTECTED),
+                        static fn (ReflectionClass $ancestor): array => $ancestor->getProperties(),
```

Comment:
Ok, but do we have an accompanying test? 

This sort of change should have raised mutations before :thinking:

Reply:
It should generate mutant here but it does not 🤷‍♂️ https://github.com/Roave/BetterReflection/actions/runs/3143133403/jobs/5107521310

The mutant appears in master https://github.com/Roave/BetterReflection/actions/runs/3144134270/jobs/5109743535

And now should be killed: https://github.com/Roave/BetterReflection/actions/runs/3144233387/jobs/5109970208